setup.py

```diff
@@ -116,7 +116,8 @@ def girder_install(self, component):
             'girder-version.json',
             'conf/girder.dist.cfg',
             'mail_templates/*.mako',
-            'mail_templates/**/*.mako'
```

Comment:
That's if they remember to do so. Our test suite runs in a git clone environment, not the packaged environment, so it would be easy for someone to forget, and then we'd hear about it at deploy time when someone with a pip-installed server is seeing 500 exceptions when their server tries to send out emails.

Basically, the cost of having this line in there as protection against that future possibility is worth it to me. The cost is one extra line of code, the benefit is avoiding this possible headache in the future. I'm happy to document that we allow mail templates in subdirectories, so that it's an officially supported thing rather than just something we happen to support.